src/app/services/property-detail/property-detail.service.ts

```diff
@@ -3,15 +3,24 @@ import { Observable } from 'rxjs';
 import { HttpService } from 'src/app/services/http.service';
 
 @Injectable({
-    providedIn: 'root'
+  providedIn: 'root'
 })
 export class PropertyDetailService {
-    constructor(
-        private http: HttpService) {
-        }
-    getProperty(slug): Observable<any> {
+  isDisplayedModal: boolean = false;
```

Comment:
I am suggesting we use
```suggestion
  isDisplayedModal = false;
```
since we get a communication from angular saying

> Type boolean trivially inferred from a boolean literal, remove type annotation

which simply means TS understands that false is a boolean